newsfragments/1924.bugfix.rst

```diff
@@ -0,0 +1 @@
+The ``TrioToken`` class can now be used as a target of a weak reference.
```

Comment:
Sorry I didn't catch this on the first round...  Note that this doesn't produce a link.

https://trio--1924.org.readthedocs.build/en/1924/history.html#bugfixes

I think this would cut it.

```suggestion
The :class:`TrioToken` class can now be used as a target of a weak reference.
```

Reply:
That wasn't quite it, but I've fixed the link now.